src/rules/utils.ts

```diff
@@ -652,6 +660,13 @@ export const isDescribe = (
   );
 };
 
+export function isTopMostJestFunction(node: TSESTree.CallExpression): boolean {
```

Comment:
I'd prefer that this be inside the rule, rather than in `utils` - iirc there is another rule doing something similar, but I'm going to be doing a spring-cleaning of the codebase this weekend, so I can refactor this into a utility method in that PR.

For now, it'll just be cleaner to have it in the rule :)